Source/WebCore/editing/cocoa/EditorCocoa.mm

```diff
@@ -102,7 +102,7 @@
     outTypesAndData.append(std::make_pair(PasteboardCustomData::cocoaType(), PasteboardCustomData { element.document().originIdentifierForPasteboard(), { } }.createSharedBuffer()));
 
     if (elementRange) {
-        if (auto archive = LegacyWebArchive::create(*elementRange)) {
+        if (auto archive = LegacyWebArchive::create(*elementRange, { })) {
```

Comment:
Why not have a default parameter value?

Reply:
I tried adding:
`WEBCORE_EXPORT static RefPtr<LegacyWebArchive> create(const SimpleRange&, ArchiveOptions&& = { });`, and got error:
`error: default member initializer for 'shouldSaveScriptsFromMemoryCache' needed within definition of enclosing class 'LegacyWebArchive' outside of member functions`.
which seems to indicate `ArchiveOptions` cannot be initialized in header.

The code below seems to work; I will update the patch to add `create*` functions without `Options` parameter.
`WEBCORE_EXPORT static RefPtr<LegacyWebArchive> create(const SimpleRange&);`
and
```
RefPtr<LegacyWebArchive> LegacyWebArchive::create(const SimpleRange& range)
{
    return LegacyWebArchive::create(range, { });
}
```